### PR TITLE
#367 Optimise Dependencies

### LINKED ISSUE #367: Reduce dependencies on newer TFM
**Describe the problem you are trying to solve.**
I do not want dependencies to be included which are not needed.

**Available workarounds**
Accept un-necessary dependency

**Additional context**
N/a

### PR DESCRIPTION
**Pull request by thompson-tomo**
**What kind of change does this PR introduce?**
update

**What is the current behavior?**
Packages are the same for all TFM's

**What is the new behavior?**
Less dependencies are included for net 6+ due to them being available as part of the framework

**What might this PR break?**
n/a

**Other information**:
closes #367

### CODE REVIEW

File: src/Fusillade/Fusillade.csproj [hunk 1]
```diff
@@ -9,7 +9,10 @@
 	<ItemGroup>
 		<PackageReference Include="punchclock" Version="3.*" />
 		<PackageReference Include="Splat" Version="14.*" />
-		<PackageReference Include="System.Net.Http" Version="4.3.4" />
 		<PackageReference Include="System.Reactive" Version="6.0.0" />
 	</ItemGroup>
+
+	<ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+		<PackageReference Include="System.Net.Http" Version="4.3.4" />
+	</ItemGroup>
```
Comment (ChrisPulman) [inline src/Fusillade/Fusillade.csproj:17]: Please remove this section

File: src/Fusillade/Fusillade.csproj [hunk 2]
```diff
@@ -9,7 +9,10 @@
 	<ItemGroup>
 		<PackageReference Include="punchclock" Version="3.*" />
 		<PackageReference Include="Splat" Version="14.*" />
-		<PackageReference Include="System.Net.Http" Version="4.3.4" />
 		<PackageReference Include="System.Reactive" Version="6.0.0" />
```
Comment (ChrisPulman) [inline src/Fusillade/Fusillade.csproj:12]: Please remove this reference

**Review by ChrisPulman — CHANGES_REQUESTED** [review]

File: src/Fusillade/Fusillade.csproj (hunk 1 above)
Comment (thompson-tomo) [reply src/Fusillade/Fusillade.csproj]: done

File: src/Fusillade/Fusillade.csproj (hunk 2 above)
Comment (thompson-tomo) [reply src/Fusillade/Fusillade.csproj]: done.

**Comment by thompson-tomo** [comment]
Have made requested changes

**Review by ChrisPulman — APPROVED** [review]

**Pull request merged**